hack/release-images.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+if [[ -z "${REGISTRY}" || -z "${REGISTRY_TAG}" || -z  "${REGISTRY_ACCOUNT}" ]]; then
+    echo "Please set all REGISTRY, REGISTRY_TAG and REGISTRY_ACCOUNT environment variables!" 
+    exit 1
+fi
+
+CONTROLLER_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-controller:${REGISTRY_TAG}
+OPERATOR_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-operator:${REGISTRY_TAG}
+MUST_GATHER_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-must-gather:${REGISTRY_TAG}
+MUST_GATHER_API_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-must-gather-api:${REGISTRY_TAG}
+UI_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-ui:${REGISTRY_TAG}
+UI_PLUGIN_IMAGE=${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-console-plugin:${REGISTRY_TAG}
```

Comment:
we won't have corresponding tags for UI images, no?

Reply:
ok, REGISTRY_TAG gets the branch name, not the last commit hash, and that should be the same